.github/workflows/continous-integration.yml

```diff
@@ -29,30 +36,23 @@ jobs:
           php-version: ${{ matrix.php }}
           extensions: json, mbstring
 
-      - name: Install Codeception dependencies
-        run: |
-          composer install --ignore-platform-reqs
-          composer require codeception/module-asserts:* \
-            codeception/util-universalframework:* \
-            codeception/module-rest:* \
-            lucatume/wp-browser:^3.1 --ignore-platform-reqs
+      - name: Install Dependencies
+        run: composer install
 
       - name: Run Codeception Tests w/ Docker.
+        # Codeception requires PHP version >= 8.0
+        # See:
+        # https://github.com/Codeception/module-asserts
+        # https://packagist.org/packages/codeception/util-universalframework
+        # https://github.com/Codeception/module-rest
+        if: ${{ matrix.php >= '8.0' }}
```

Comment:
If we can drop support for at least `7.3` and `7.4` for test runs, then we don't need this conditional anymore.

Reply:
True